### PR TITLE
Replace infura with alchemy

### PR DESCRIPTION
**Pull request by sshmm**
## What
replacing infura rpcs and keys with alchemy
the following keys are used
REACT_APP_ETH_MAINNET_API_KEY=ethereum_mainnet_api_key_from_alchemy
REACT_APP_ETH_GOERLI_API_KEY=ethereum_goerli_api_key_from_alchemy
REACT_APP_OPTIMISM_RPC_URL=optimism_mainnet_api_key_from_alchemy
REACT_APP_OPTIMISM_GOERLI_RPC_URL=optimism_goerli_api_key_from_alchemy

make sure that they are holding the keys not the whole link and are available for production before testing
Circle CI are passing after passing the correct keys.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -59,10 +58,13 @@ TEST_ON_HARDHAT_NODE=
...
 OPTIMISM_GOERLI_RPC_URL=get_a_rpc_from_alchemy
-REACT_APP_OPTIMISM_GOERLI_RPC_URL=get_a_rpc_from_alchemy_for_magic
+REACT_APP_ETH_MAINNET_API_KEY=get_ethereum_mainnet_api_key_from_alchemy
+REACT_APP_ETH_GOERLI_API_KEY=get_ethereum_goerli_api_key_from_alchemy
+REACT_APP_OPTIMISM_MAINNET_API_KEY=get_optimism_mainnet_api_key_from_alchemy
+REACT_APP_OPTIMISM_GOERLI_API_KEY=get_optimism_goerli_api_key_from_alchemy
 
 # used to fork mainnet when testing
-HARDHAT_ARCHIVE_RPC_URL=https://mainnet.infura.io/v3/your_infura_project_id
+HARDHAT_ARCHIVE_RPC_URL=get_ethereum_mainnet_rpc_url_from_alchemy
```
Comment (crabsinger) [inline .env.example:67]: we can just remove this completely now and just use the value of REACT_APP_ETH_MAINNET_API_KEY interpolated into a full right?

Comment (sshmm) [reply .env.example]: I made the necessary changes to use REACT_APP_ETH_MAINNET_API_KEY instead

File: .circleci/continue-config.yml [hunk 2]
```diff
@@ -81,6 +78,9 @@ jobs:
           pkg-manager: yarn
       - run: yarn cypress install
       - run: yarn cypress verify
+      - run:
+          name: 'echo REACT_APP_ETH_MAINNET_API_KEY from CI context'
+          command: echo $REACT_APP_ETH_MAINNET_API_KEY
```
Comment (crabsinger) [inline .circleci/continue-config.yml:83]: all these are just for debugging and can now be removed, right? @sshmm

Comment (sshmm) [reply .circleci/continue-config.yml:83]: @crabsinger  as far as I know, it's necessary to echo the variables to make them visible to the rest of the scripts.

Comment (crabsinger) [reply .circleci/continue-config.yml:83]: @sshmm hmm, im very surprised by that. have you tried removing this echo?

Comment (sshmm) [reply .circleci/continue-config.yml]: > @sshmm hmm, im very surprised by that. have you tried removing this echo?

@crabsinger  you are right. it's not required.

Comment (sshmm) [reply .circleci/continue-config.yml]: @crabsinger  I have removed echo commands and rebased the branch

**Review by crabsinger — APPROVED** [review]

**Pull request merged**